packages/wallet/src/redux/protocols/direct-funding/actions.ts

```diff
@@ -1,36 +1,20 @@
 import * as actions from '../../actions';
-import { ActionConstructor } from '../../utils';
-import { TwoPartyPlayerIndex } from '../../types';
 import { isCommonAction, EmbeddedProtocol, routerFactory } from '../../../communication';
 
 // -------
 // Actions
 // -------
-export interface DirectFundingRequested {
```

Comment:
I think that would make sense but should be unrelated to this change. Here I just switched `initialize` in `directFunding` to take named arguments instead of an action (so it follows the same pattern as other reducers)

Reply:
This comment was related to the fact that the `DirectFundingRequested` action was deleted.

Reply:
`DirectFundingRequested` was only used to pass arguments into `direct-funding`'s `initialize` .